config/config_test.go

```diff
@@ -915,6 +915,88 @@ func TestRequestLimitsValidateAndLog(t *testing.T) {
 	}
 }
 
+func TestRequestLogging(t *testing.T) {
+	hook := testLogrus.NewGlobal()
+
+	type logComponents struct {
+		msg string
+		lvl logrus.Level
+	}
+
+	testCases := []struct {
+		desc                string
+		inRequestLoggingCfg *RequestLogging
+		expectedLogInfo     []logComponents
+	}{
+		{
+			desc: "negative sampling rate, expect fatal log",
```

Comment:
Please use name instead of description. Names should be simple with no spaces. If needed, use comments for more detail. For example:

```
name: "invalid_negative" // must be greater or equal to 0. expect fatal log.
name: "invalid_high" // must be less than or equal to 1. expect fatal log.
name: "valid_one"
name: "valid_zero"
name: "valid"
```

Reply:
I forgot about `t.Run()`. Modified